src/lib/isIBAN.js

```diff
@@ -67,7 +67,7 @@ const ibanRegexThroughCountryCode = {
   NO: /^(NO[0-9]{2})\d{11}$/,
   PK: /^(PK[0-9]{2})[A-Z0-9]{4}\d{16}$/,
   PL: /^(PL[0-9]{2})\d{24}$/,
-  PS: /^(PS[0-9]{2})[A-Z0-9]{4}\d{21}$/,
+  PS: /^(PS[0-9]{2})[A-Z]{4}\d{21}$/,
```

Comment:
Kindly correct me, but If I understand this correctly, the Palestine RegExp would be wrong here (the last part):

On page 6 of the supplied PDF we can see the convention:
![grafik](https://github.com/user-attachments/assets/9c11dab2-5a1b-4cc7-89bb-68ad034d5996)

The supplied PDF says the IBAN structure is `PS2!n4!a21!c`
The regex here shows `/^(PS[0-9]{2})[A-Z]{4}\d{21}$/`

So I would argue, that the `\d{21}` part should actually be ` [a-zA-Z0-9]` or `[A-Z0-9]` (since we don't care about case, I think).

what do you think?